modules/layer2.js

```diff
@@ -0,0 +1,23 @@
+var Layer2 = `<p id="name">Cuppy</p>
```

Comment:
Your modules should be functions.

I'll recommend you to watch the course I shared. That'll explain a lot of things.

Reply:
okay sir